lib/charms/haproxy/v1/haproxy_route.py

```diff
@@ -467,6 +467,7 @@ class RequirerApplicationData(_DatabagModel):
     Attributes:
         service: Name of the service requesting HAProxy routing.
         ports: List of port numbers on which the service is listening.
+        hosts: List of backend server addresses. Currently only support IP addresses.
```

Comment:
"Currently only support IP addresses." is not a lib limitation, is it? I don't think we should state that here

Reply:
makes sense, i removed it.